vendor.conf

```diff
@@ -47,50 +47,50 @@ go.etcd.io/bbolt                                    v1.3.5
 google.golang.org/grpc                              v1.27.1
 gotest.tools/v3                                     v3.0.2
 
 # cgroups dependencies
-github.com/cilium/ebpf                              4032b1d8aae306b7bb94a2a11002932caf88c644
+github.com/cilium/ebpf                              1c8d4c9ef7759622653a1d319284a44652333b28
 
 # cri dependencies
-github.com/containerd/cri                           4f8a580795344b0f4c1146a3abce0409962f3890 # master
+github.com/containerd/cri                           aa0f4fd37b82a273156d35a7e13831bd0940ab99 # master
 github.com/davecgh/go-spew                          v1.1.1
 github.com/docker/docker                            4634ce647cf2ce2c6031129ccd109e557244986f
 github.com/docker/spdystream                        449fdfce4d962303d702fec724ef0ad181c92528
 github.com/emicklei/go-restful                      v2.9.5
+github.com/go-logr/logr                             v0.2.0
 github.com/google/gofuzz                            v1.1.0
-github.com/json-iterator/go                         v1.1.8
+github.com/json-iterator/go                         v1.1.9
 github.com/modern-go/concurrent                     1.0.3
 github.com/modern-go/reflect2                       v1.0.1
-github.com/opencontainers/selinux                   bb88c45a3863dc4c38320d71b890bb30ef9feba4
+github.com/opencontainers/selinux                   v1.5.1
 github.com/seccomp/libseccomp-golang                v0.9.1
-github.com/stretchr/testify                         v1.4.0
 github.com/tchap/go-patricia                        v2.2.6
 golang.org/x/crypto                                 bac4c82f69751a6dd76e702d54b3ceb88adab236
-golang.org/x/oauth2                                 0f29369cfe4552d0e4bcddc57cc75f4d7e672a33
-golang.org/x/time                                   9d24e82272b4f38b78bc8cff74fa936d31ccd8ef
+golang.org/x/oauth2                                 858c2ad4c8b6c5d10852cb89079f6ca1c7309787
+golang.org/x/time                                   555d28b269f0569763d25dbe1a237ae74c6bcc82
 gopkg.in/inf.v0                                     v0.9.1
 gopkg.in/yaml.v2                                    v2.2.8
-k8s.io/api                                          v0.18.2
-k8s.io/apimachinery                                 v0.18.2
-k8s.io/apiserver                                    v0.18.2
-k8s.io/client-go                                    v0.18.2
-k8s.io/cri-api                                      v0.18.2
-k8s.io/klog                                         v1.0.0
+k8s.io/api                                          v0.19.0-beta.2
+k8s.io/apimachinery                                 v0.19.0-beta.2
+k8s.io/apiserver                                    v0.19.0-beta.2
+k8s.io/client-go                                    v0.19.0-beta.2
+k8s.io/cri-api                                      v0.19.0-beta.2
+k8s.io/klog/v2                                      v2.2.0
 k8s.io/kubernetes                                   v1.18.2
```

Comment:
cri no longer uses k8s.io/kubernetes guy named dims dropped it :-) If containerd still needs it for something we should move to containerd's dependency area and change to v1.19.0-beta.2 ... if it doesn't need it should be ok to remove.

Reply:
whoops i forgot to drop this :)